src/server.tsx

```diff
@@ -44,34 +45,45 @@ interface Response {
     css: string;
     js: string;
     meta: EpicTestTracking;
+    debug?: Debug;
 }
 
 const [, fetchConfiguredEpicTestsCached] = cacheAsync(fetchConfiguredEpicTests, 60);
 
-const asResponse = (component: JsComponent, meta: EpicTestTracking): Response => {
+const asResponse = (component: JsComponent, meta: EpicTestTracking, debug?: Debug): Response => {
     const { el, js } = component;
     const { html, css } = extractCritical(renderToStaticMarkup(el));
-    return { html, css, js, meta };
+    return { html, css, js, meta, debug };
 };
 
 const buildEpic = async (
     pageTracking: EpicPageTracking,
     targeting: EpicTargeting,
+    params: Params,
 ): Promise<Response | null> => {
     const configuredTests = await fetchConfiguredEpicTestsCached();
     const hardcodedTests = await getAllHardcodedTests();
     const tests = [...configuredTests.tests, ...hardcodedTests];
-    const result = findTestAndVariant(tests, targeting);
+
+    let result: Result;
+
+    if (params.force) {
```

Comment:
I think the issue with this logic is that we can't combine `?debug` and `?force`. Forcing a test/variant bypasses the debug logic. Not sure how useful that would be but thought it was worth a mention.

Reply:
I _think_ this seems right - "Forcing a test/variant bypasses the debug logic" - since forcing bypasses any filtering that might happen.

Reply:
Yeah I agree, it makes sense. It's more of a general question, is there any value in trying to provide debug info when forcing? Perhaps doing the filtering anyway but returning the forced test/variant? I agree it may not be worth the added complexity but as a user I sort of expected that (even though I don't have a use case in mind).